aptos-move/parallel-executor/src/proptest_types/types.rs

```diff
@@ -59,6 +61,31 @@ impl<K: Hash + Clone + Debug + Eq + PartialOrd> ModulePath for KeyType<K> {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Arbitrary)]
+pub struct ValueType<V: Into<Vec<u8>> + Debug + Clone + Eq + Arbitrary>(
+    /// Wrapping the types used for testing to add TransactionWrite trait implementation (below).
+    pub V,
+);
+
+impl<V: Into<Vec<u8>> + Debug + Clone + Eq + Send + Sync + Arbitrary> TransactionWrite
+    for ValueType<V>
+{
+    fn extract_raw_bytes(&self) -> Option<Vec<u8>> {
+        Some(self.0.clone().into())
+    }
+}
+
+impl<V: Into<Vec<u8>> + Debug + Clone + Eq + Send + Sync + Arbitrary> ValueType<V> {
+    fn _extract_u128(&self) -> Option<u128> {
```

Comment:
what is this, remove?

Reply:
We will use it and make it non "_" with PE. logically belongs here I thought.